cypress/integration/vanilla-opensearch-dashboards/dashboard_sample_data_spec.js

```diff
@@ -0,0 +1,104 @@
+import { TestFixtureHandler, CommonUI, DashboardPage, MiscUtils } from '@opensearch-dashboards-test/opensearch-dashboards-test-library'
+
+/**
+ * dashboard_sample_data test suite description:
+ * 1) Visit the home page of opensearchdashboard, check key UI elements display
+ * 2) add sample data of eCommerce, flights, web logs from tutorial page
+ * 3) check each sample data dashboard key UI elements display
+ */
+
+const testFixtureHandler = new TestFixtureHandler(cy, Cypress.env('openSearchUrl'))
+const commonUI = new CommonUI(cy)
+const dashboardPage = new DashboardPage(cy)
+const miscUtils = new MiscUtils(cy)
+describe('dashboard sample data validation', () => {
+    before(() => {
+
+    })
+
+    after(() => {
+
+    })
+
+    describe('checking home page', () => {
+        before(() => {
+            // Go to the home page
+            miscUtils.visitPage('app/home#')
+        })
+
+        after(() => {
+
+        })
+
+        it('checking opensearch_dashboards_overview display', () => {
+            // Check that opensearch_dashboards_overview is visable
+            commonUI.checkElementExists('a[href="/_dashboards/app/opensearch_dashboards_overview"]', 1)
```

Comment:
not all dashboards instance will set `server.basePath: /_dashboards`
could we extract the `_dashboards` to basePath parameter as part of test run parameter?

Reply:
will make the change to address this.